dotnet/src/Connectors/Connectors.AI.PaLM/TextCompletion/TextCompletionResponse.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.PaLM.TextCompletion;
+
+/*
+/// <summary>
+/// HTTP Schema for completion response.
+/// </summary>
+public sealed class TextCompletionResponse
+{
+    /// <summary>
+    /// Completed text.
+    /// </summary>
+    [JsonPropertyName("generated_text")]
+    public string? Text { get; set; }
+}*/
+
+public class TextCompletionResponse
+{
+    public Candidate[] candidates { get; set; }
```

Comment:
Pascal case is typically used for property names in C#, so please capitalize first later of each property of data contracts in this and other files.

Reply:
fixed